### PR TITLE
Implements Analytics on Upgrade Modal pops

### PR DESCRIPTION
**Pull request by discorick**
References: #179 

image

```json
{
  "anonymousId": "f9275b47-dbb6-4bba-b3f8-db5f871dbefc",
  "category": null,
  "context": {
    "page": {
      "path": "/settings/discorick/upgrade",
      "referrer": "http://huboard.rails/",
      "search": "",
      "title": "HuBoard",
      "url": "/settings/discorick/upgrade"
    },
    "userAgent": "blahblahblah",
    "library": {
      "name": "analytics.js",
      "version": "2.11.1"
    },
    "ip": "68.146.230.217"
  },
  "integrations": {},
  "messageId": "ajs-64cab6bf812c0339964dca3c66218775",
  "name": null,
  "properties": {
    "url": "/settings/discorick/upgrade",
    "path": "/settings/discorick/upgrade",
    "referrer": "http://huboard.rails/",
    "search": "",
    "title": "HuBoard"
  },
  "receivedAt": "2016-03-29T17:13:04.650Z",
  "sentAt": "2016-03-29T17:13:05.708Z",
  "timestamp": "2016-03-29T17:13:04.649Z",
  "type": "page",
  "userId": "1130665",
  "originalTimestamp": "2016-03-29T17:13:05.707Z"
}
```

### CODE REVIEW

File: ember-accounts/app/mixins/analytics.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+//A temporary solution to get GA shipped, Really what I want here
+//is conventionally queued Analytics events - @discorick
+export default Ember.Mixin.create({
+  actions: {
+    purchase(model) {
+      window.analytics.page({
+        url: `/settings/${this.get('model.login')}/upgrade`,
```
Comment (dahlbyk) [inline ember-accounts/app/mixins/analytics.js]: Technically shouldn't `url` include the full path? Not sure if that matters for GA integration. ([Docs](https://segment.com/docs/libraries/analytics.js/#page))

Comment (discorick) [inline ember-accounts/app/mixins/analytics.js]: In all our other server-side `page()`'s we subbing a relative path for the full url.

Comment (dahlbyk) [inline ember-accounts/app/mixins/analytics.js]: Confirmed. Guess it doesn't matter. :grinning: 

Next question: thoughts on setting `referrer` here to `window.location.toString()`?

Comment (discorick) [inline ember-accounts/app/mixins/analytics.js]: I thought about it, honestly I omitted it to match all the other payloads to keep things looking consistent.

Comment (dahlbyk) [inline ember-accounts/app/mixins/analytics.js]: :+1:

**Comment by dahlbyk** [comment]
:clap:

**Pull request merged**